scripts/setup-k8s-master.sh.in

```diff
@@ -53,6 +54,13 @@ apiServerCertSANs:
 - ${LB_DNS}
 EOF
 
+if [[ "{{NetworkingProvider}}" == "calico" ]]; then
+cat >>/tmp/kubeadm.yaml <<EOF
+networking:
+  podSubnet: 192.168.0.0/16
```

Comment:
Should this _always_ be set, regardless of CNI plugin?

Reply:
@alexbrand No, for example it should not be set for weave and flannel requires a different value to be set: https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network

I still need to verify if we need to set the sysctl for net.bridge.bridge-nf-call-iptables when configuring weave, as that page suggests.